src/notify.js

```diff
@@ -28,6 +28,8 @@ function Notify(title, options = {}) {
         notifyError = null,
         closeOnClick = false,
         timeout = null,
+        swUrl = null,
```

Comment:
As suggested, I think this use method is likely not one most people will seek to use. I think we should remove this for the time being, and concentrate on a solid implementation of triggering notifications from within a SW. We can always add it later if there is a demand for it.